src/main/java/org/apache/maven/plugin/compiler/TestCompilerMojo.java

```diff
@@ -27,11 +27,13 @@
 import java.util.Collection;
 import java.util.Collections;
 import java.util.HashSet;
+import java.util.Iterator;
```

Comment:
Please fix this breaking issue: 
`Error:  src/main/java/org/apache/maven/plugin/compiler/TestCompilerMojo.java:[30,8] (imports) UnusedImports: Unused import - java.util.Iterator.`

Reply:
Done.